tests/unit/client/feedback/integrations/huggingface/test_dataset.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Any, Dict
+
+import pytest
+from argilla.client.feedback.dataset.local.dataset import FeedbackDataset
+from argilla.client.feedback.integrations.huggingface.dataset import HuggingFaceDatasetMixin
+from argilla.client.feedback.schemas.fields import TextField
+from argilla.client.feedback.schemas.questions import TextQuestion
+from argilla.client.feedback.schemas.records import FeedbackRecord
+
+
+class TestSuiteHuggingFaceDatasetMixin:
+    @pytest.mark.parametrize(
+        "record, hf_record",
+        [
+            (FeedbackRecord(fields={"required-field": "value"}), {"required-field": "value", "optional-field": None}),
```

Comment:
```suggestion
            (FeedbackRecord(fields={"required-field": "value"}), {"required-field": "value", "optional-field": None}),
            (FeedbackRecord(fields={"required-field": "value", "optional-field": None}), {"required-field": "value", "optional-field": None}),
```

Reply:
This is not needed because what we want to test here is that either the value is propagated to the `datasets.Dataset` or that `None` is set if not providing, so setting `None` is the same as setting any other value